src/utils/db/db.ts

```diff
@@ -422,7 +423,27 @@ export class DB {
             await client.query('DELETE FROM posthog_person WHERE id = $1', [person.id])
         })
         if (this.clickhouse) {
-            await this.clickhouseQuery(`ALTER TABLE person DELETE WHERE id = '${escapeClickHouseString(person.uuid)}'`)
+
+            if (this.kafkaProducer) {
+                this.kafkaProducer.queueMessage({
+                    topic: KAFKA_PERSON,
+                    messages: [
+                        {
+                            value: Buffer.from(
+                                JSON.stringify({
+                                    created_at: castTimestampOrNow(person.created_at, TimestampFormat.ClickHouse),
+                                    properties: JSON.stringify(person.properties),
+                                    team_id: person.team_id,
+                                    is_identified: person.is_identified,
+                                    id: person.uuid,
+                                    deleted: 1
```

Comment:
I'd make this `is_deleted` to conform with how most schemas do soft deletes. Also to conform with `is_identified` in this table. Just for consistency.